tests/tests_suite_test.go

```diff
@@ -30,81 +31,224 @@ import (
 )
 
 const (
-	// TODO - maybe randomize namespace
-	testNamespace             = "ssp-operator-functests"
-	commonTemplatesTestNS     = "ssp-operator-functests-templates"
-	timeout                   = 8 * time.Minute
-	templateValidatorReplicas = 1
+	envExistingCrName        = "TEST_EXISTING_CR_NAME"
+	envExistingCrNamespace   = "TEST_EXISTING_CR_NAMESPACE"
+	envSkipUpdateSspTests    = "SKIP_UPDATE_SSP_TESTS"
+	envSkipCleanupAfterTests = "SKIP_CLEANUP_AFTER_TESTS"
+
+	timeout = 10 * time.Minute
```

Comment:
Well, who knows how fast is the testing cluster. Sometimes even 8 minutes was not enough.

If the timeout hits, it is probably some error that will cause the function `waitUntilDeployed()` to timeout too, and all remaining tests will immediately fail. It would be slow if all 58 test cases would need to wait for 10 minutes.